src/app/workflow/executions/executions-tab.component.ts

```diff
@@ -71,42 +70,43 @@ export class ExecutionsTabComponent extends EntryTab implements OnChanges {
   constructor(
     private extendedGA4GHService: ExtendedGA4GHService,
     private alertService: AlertService,
-    protected sessionQuery: SessionQuery,
-    private checkerWorkflowQuery: CheckerWorkflowQuery
+    protected sessionQuery: SessionQuery
   ) {
     super();
   }
 
   ngOnChanges() {
-    this.alertService.start('Retrieving metrics data');
-    this.trsID = this.checkerWorkflowQuery.getTRSId(this.entry);
-    this.extendedGA4GHService
-      .aggregatedMetricsGet(this.trsID, this.version.name)
-      .pipe(takeUntil(this.ngUnsubscribe))
-      .subscribe(
-        (metrics) => {
-          this.resetMetricsData();
-          if (metrics) {
-            for (const [partner, metric] of Object.entries(metrics)) {
-              this.setMetricsObject(this.metrics, partner, metric);
-            }
-            this.partners = Array.from(this.metrics.keys());
-            this.metricsExist = this.partners.length > 0;
-            if (this.metricsExist) {
-              // Display metrics for ALL platforms
-              const platform =
-                this.partners.filter((partner) => partner === PartnerEnum.ALL).length === 1
-                  ? this.partners.filter((partner) => partner === PartnerEnum.ALL)[0]
-                  : this.partners[0];
-              this.selectPartner(platform);
+    this.resetMetricsData();
+    if (this.version) {
```

Comment:
Added this version check because it's possible for there to be no version (if it's a hosted workflow for example). Without this check, there's a console error saying that version is null